core/services/keeper/registry_synchronizer_log_listener.go

```diff
@@ -9,10 +9,6 @@ import (
 	"github.com/smartcontractkit/chainlink/core/store/models"
 )
 
-func (rs *RegistrySynchronizer) OnConnect() {}
-
-func (rs *RegistrySynchronizer) OnDisconnect() {}
```

Comment:
I look forward to the day that run manager v1 is removed and we can axe these callbacks forever.

Reply:
:+1: axe 'em!